src/modules/site-v2/templates/tools/heritability_calculator/result.html

```diff
@@ -322,22 +320,22 @@
         doc.fromHTML('The broad-sense heritability is the amount of trait variance that comes from genetic differences in the assayed group ', 20, 85, {'align': 'justify', 'maxWidth': doc.internal.pageSize.getWidth()-40},function(){},margins)
         doc.fromHTML('of strains. Generally, it is the ratio of genetic variance to total (genetic plus environmental) variance.', 20, 100, {'align': 'justify', 'maxWidth': doc.internal.pageSize.getWidth()-40},function(){},margins)
 
-        var a = document.createElement('a');
-            var pdfblob = new Blob([ doc.output('blob') ], { type : 'application/pdf'});
-            a.href = window.URL.createObjectURL(pdfblob);
-            a.download = "{{ fname }}.pdf";
-            a.click();
-        })
-        image.src = url;
+        const a = document.createElement('a');
+        const pdfblob = new Blob([ doc.output('blob') ], { type : 'application/pdf'});
+        a.href = window.URL.createObjectURL(pdfblob);
+        a.download = "{{ fname }}.pdf";
+        a.click();
+      })
+      image.src = url;
     }
 
-    var getPDF = function(){
+    function getPDF() {
       loadCanvas();
     }
 
-    $(document).ready(function(){
-      if (chartJson.length != 0){
-        var node= document.getElementById("svgchartarea");
+    $(document).ready(function() {
```

Comment:
unless we rely on jQuery, we can use standards: window.addEventListener('load', () => {}, false);